tests/p_libxml2-python/0-install-libxml2-python.sh

```diff
@@ -5,7 +5,7 @@
 t_Log "Running $0 - installing libxml2-python."
 
 if [ "$centos_ver" -ge 8 ] ; then
-t_InstallPackage python36 python3-libxml2
```

Comment:
Do we need to install *some* version of python3? Does this work like this on EL8?

Reply:
Let me check.

Reply:
Ok, forgot to push a commit where I reverted that, will include it when I will change those `-ge` flags.